docker-compose.yml

```diff
@@ -56,6 +56,12 @@ services:
     container_name: lantern-mq
     image: rabbitmq:3-management
 
+  lantern_front_end:
+    container_name: lantern_front_end
+    image: onchealthit/lantern-front-end
```

Comment:
```suggestion
    image: onchealthit/lantern-front-end
    environment:
      - LANTERN_<frontend url>=<frontend url>
```

Reply:
I think the issue here was actually that the image on the docker-hub repo needed to be updated, so im waiting for confirmation that that was the fix!